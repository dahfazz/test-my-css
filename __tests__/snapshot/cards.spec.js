var sass = require('node-sass');

it('Cards snapshot is OK', async () => {
  
  var result = sass.renderSync({
    data: `
    @import 'src/scss/variables';
    @import 'src/scss/card';
    `,
    outputStyle: 'nested'
  });
  expect(result.css.toString()).toMatchSnapshot();
});