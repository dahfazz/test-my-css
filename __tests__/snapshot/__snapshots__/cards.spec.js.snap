// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards snapshot is OK 1`] = `
".card {
  margin-left: 1rem;
  border: 8px solid black; }
  .dark .card {
    border-color: white; }
"
`;
